site/en/getstarted/cluster/install_cluster-docker.md

```diff
@@ -17,6 +17,9 @@ summary: Learn how to install Milvus cluster with Docker Compose.
 Docker-compose cannot deploy Milvus distributed clusters across machines, and can only be used in test environments. It is not recommended that users deploy Milvus distributed clusters in this way in production environments.
```

Comment:
Should be "Docker Compose" rather than "Docker-compose".

Reply:
[db73a8c](https://github.com/milvus-io/milvus-docs/pull/1624/commits/db73a8c4bbda9a37898272ddebfcc4506c94e5ae)